manager/integration/tests/test_rwx.py

```diff
@@ -572,4 +574,63 @@ def test_rwx_offline_expansion(): # NOQA
     And
     - 1.5 Gi of data is successfully written to the expanded volume
     """
-    pass
+    pvc_name = 'pvc-deployment-rwx-expand-test'
+    pvc['metadata']['name'] = pvc_name
+    pvc['spec']['storageClassName'] = 'longhorn'
+    pvc['spec']['accessModes'] = ['ReadWriteMany']
+    pvc['spec']['resources']['requests']['storage'] = str(1 * Gi)
+
+    core_api.create_namespaced_persistent_volume_claim(
+        body=pvc, namespace='default')
+
+    deployment = make_deployment_with_pvc(
+        'deployment-rwx-expand-test', pvc_name, replicas=1)
+    apps_api = get_apps_api_client()
+    create_and_wait_deployment(apps_api, deployment)
+
+    pv_name = get_volume_name(core_api, pvc_name)
+    deployment_label_selector = "name=" + \
+                                deployment["metadata"]["labels"]["name"]
+
+    deployment_pod_list = \
+        core_api.list_namespaced_pod(namespace="default",
+                                     label_selector=deployment_label_selector)
+
+    pod_name = deployment_pod_list.items[0].metadata.name
+    test_data = generate_random_data(VOLUME_RWTEST_SIZE)
+    write_pod_volume_data(core_api, pod_name, test_data, filename='test')
+
+    deployment['spec']['replicas'] = 0
+    apps_api.patch_namespaced_deployment(body=deployment,
+                                         namespace='default',
+                                         name=deployment["metadata"]["name"])
+
+    wait_for_volume_detached(client, pv_name)
+    expand_and_wait_for_pvc(core_api, pvc, EXPANDED_VOLUME_SIZE*Gi)
+    wait_for_volume_expansion(client, pv_name)
+    wait_for_volume_detached(client, pv_name)
+
+    deployment['spec']['replicas'] = 1
+    apps_api.patch_namespaced_deployment(body=deployment,
+                                         namespace='default',
+                                         name=deployment["metadata"]["name"])
+    wait_deployment_replica_ready(apps_api, deployment["metadata"]["name"], 1)
+    wait_for_volume_healthy(client, pv_name)
+
+    deployment_pod_list = \
+        core_api.list_namespaced_pod(namespace="default",
+                                     label_selector=deployment_label_selector)
+    pod_name = deployment_pod_list.items[0].metadata.name
+
+    # check data written before expansion
+    resp = read_volume_data(core_api, pod_name, "test")
+    assert resp == test_data
+
+    data_size_in_mb = 1536
+    write_pod_volume_random_data(core_api, pod_name, "/data/test2",
+                                 data_size_in_mb)
+    command = 'stat -c \"%s\" /data/test2'
+    data_size_in_pod = exec_command_in_pod(core_api, command,
+                                           pod_name,
+                                           'default')
+    assert int(data_size_in_pod)/1024/1024 == data_size_in_mb
```

Comment:
There is some implicit info here. I would suggest making them clear in the future. 
- `write_pod_volume_random_data` is overwrite instead of resuming writing.
- the second write size is larger than 1G